firefox-ios/Client/Frontend/Autofill/AutoFillFooterView.swift

```diff
@@ -0,0 +1,71 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+
+struct AutoFillFooterView: View {
```

Comment:
I'm sorry for the nit, `AutoFill` should be `Autofill` just to be consistent with the rest of the codebase and easier to search.